README.md

```diff
@@ -207,31 +162,131 @@ Users want to determine several types of the formatting width as following
+```
+
+### Constructor
+
+#### Syntax
+
+```javascript
+new Intl.DurationFormat(locales, options)
 ```
+
+#### Parameters
+
+* `locales: Array<string> | string`: A locale string or a list of locale strings in decreasing order of preference.
+* `options?: object`: An object for configuring the behavior of the instance. It may have some or all of the following properties:
+  * `localeMatcher: "best-fit" | "lookup"`: A string denoting which locale matching algorithm to use. Defaults to `"best fit"`.
+  * `numberingSystem: string`: A string containing the name of the numbering system to be used for number formatting.
+  * `style: "long" | "short" | "narrow"`: The base style to be used for formatting. This can be overriden per-unit by setting the more granular options. Defaults to `"short"`.
+  * `years: "long" | "short" | "narrow"`: The style to be used for formatting years.
+  * `yearsDisplay: "always" | "auto"`: Whether to always display years, or only if nonzero.
+  * `months: "long" | "short" | "narrow"`: The style to be used for formatting months.
+  * `monthsDisplay: "always" | "auto"`: Whether to always display months, or only if nonzero.
+  * `weeks: "long" | "short" | "narrow"`: The style to be used for formatting weeks.
+  * `weeksDisplay: "always" | "auto"`: Whether to always display weeks, or only if nonzero.
+  * `days: "long" | "short" | "narrow"`: The style to be used for formatting days.
+  * `daysDisplay: "always" | "auto"`: Whether to always display days, or only if nonzero.
+  * `hours: "long" | "short" | "narrow" | "numeric" | "2-digit"`: The style to be used for formatting hours.
+  * `hoursDisplay: "always" | "auto"`: Whether to always display hours, or only if nonzero.
+  * `minutes: "long" | "short" | "narrow" | "numeric" | "2-digit"`: The style to be used for formatting minutes.
+  * `minutesDisplay: "always" | "auto"`: Whether to always display minutes, or only if nonzero.
+  * `seconds: "long" | "short" | "narrow" | "numeric" | "2-digit"`: The style to be used for formatting seconds.
+  * `secondsDisplay: "always" | "auto"`: Whether to always display seconds, or only if nonzero.
+  * `milliseconds: "long" | "short" | "narrow" | "numeric"`: The style to be used for formatting milliseconds.
+  * `millisecondsDisplay: "always" | "auto"`: Whether to always display milliseconds, or only if nonzero.
+  * `microseconds: "long" | "short" | "narrow" | "numeric"`: The style to be used for formatting microseconds.
+  * `microsecondsDisplay: "always" | "auto"`: Whether to always display microseconds, or only if nonzero.
+  * `nanoseconds: "long" | "short" | "narrow" | "numeric"`: The style to be used for formatting nanoseconds.
+  * `nanosecondsDisplay: "always" | "auto"`: Whether to always display nanoseconds, or only if nonzero.
+  * `fractionalDigits: number`: The number of fractional digits to be included in the output, applies automatically to the entire set of consecutive time units with style `"numeric"` at the end.
```

Comment:
I find this text pretty confusing. Is there a clearer way to express this?  Also need to describe the default behavior. Here's a start:

```suggestion
  * `fractionalDigits: number`: How many fractional digits to display in the output. 
    Additional decimal places will be truncated towards zero.
    (`Temporal.Duration.prototype.round` can be used to obtain different rounding behavior.)
    Normally this option applies to fractional seconds, but this option actually applies to the largest seconds-or-smaller unit that uses the `"numeric"` or `"2-digit"` style.
    For example, if options are `{ seconds: "narrow", milliseconds: "narrow", fractionalDigits: 2}` then the output can be `"5s 2.39ms"`.
    If this option is omitted, only nonzero decimals will be displayed and trailing zeroes will be omitted.
```